src/language/parser.js

```diff
@@ -1404,6 +1417,22 @@ class Parser {
     }
   }
 
+  // Add the comment token to Parser
```

Comment:
Developers are capable of reading function names no need to duplicate it inside comments :)